config.json

```diff
@@ -381,6 +381,20 @@
         "sorting",
         "trees"
       ]
+    },
+    {
+      "slug": "perfect-numbers",
+      "uuid": "db9eb481-a2b5-4b53-9825-a20797f930de",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
I suggest moving it so it is unlocked by armstrong-numbers

Reply:
@amscotti This can be reverted. It can be unlocked by difference-of-squares

Reply:
@Stargator Update